tests/test_wrapper.py

```diff
@@ -38,9 +38,9 @@ def setUp(self):
         self.mock_wrapper_lambda_stats = patcher.start()
         self.addCleanup(patcher.stop)
 
-        patcher = patch("datadog_lambda.metric.lambda_metric")
-        self.mock_lambda_metric = patcher.start()
-        self.addCleanup(patcher.stop)
+        # patcher = patch("datadog_lambda.metric.lambda_metric")
+        # self.mock_lambda_metric = patcher.start()
+        # self.addCleanup(patcher.stop)
```

Comment:
What's this?

Reply:
We were patching the datadog_lambda.metric.lambda_metric out for testing, but the new code uses it internally, so I unpatched it. I'll remove this snippet.